mohschema.py

```diff
@@ -24,7 +24,7 @@ def __str__(self):
 def warn(message):
     message = ">".join(STACK_LOCATION) + ": " + message
     VALIDATION_MESSAGES.append(f"{message}")
-    raise MoHValidationError(message)
+    #raise MoHValidationError(message)
```

Comment:
Should this be commented out?

Reply:
Yeah, I think it would be better to collate the warnings together and report them at the end, rather than error out when you reach any of them.